proposals/coredns.adoc

```diff
@@ -0,0 +1,82 @@
+== CoreDNS Proposal
+
+*Name of project:* CoreDNS
+
+*Description*
+
+CoreDNS is a fast, flexible DNS server that is designed to be performant with a flexible implementation. This flexibility allows CoreDNS to be easily extended to support various data sources and to implement rich DNS service behaviors. For example, response caching, query rewrite, load-balancing, zone transfer and signing.
+CoreDNS is the successor of SkyDNS.
+
+*Sponsor / Advisor from TOC*: Jonathan Boulle
+
+*Unique Identifier*: coredns
+
+*License*: Apache License v2.0
+
+*Source control repositories*: https://github.com/miekg/coredns
+
+*Initial Committers*:
+
+* Michael Richmond github: mrichmon
+* github: splack
+* Felix Cantournet github: fcantournet
+* github: leelynne
+* Matt Layher github: mdlayher
+* Vasily Vailyev github: pixelbender
+
+*Infrastructure requirements (CI / CNCF Cluster)*: N/A
+
+*Issue tracker*: https://github.com/miekg/coredns
+
+*Website*: https://coredns.io
+
+*Release methodology and mechanics*: github master
```

Comment:
Flesh this out a bit. It looks like there are no releases so far - but perhaps talk about why, and what process will be introduced (semver? prebuilt binaries?)